src/herder/test/HerderTests.cpp

```diff
@@ -3704,25 +3725,29 @@ TEST_CASE("accept soroban txs after network upgrade")
                 .getLastClosedLedgerHeader()
                 .header.ledgerVersion ==
             static_cast<uint32_t>(SOROBAN_PROTOCOL_VERSION));
-
+    for (auto node : nodes)
+    {
+        overrideSorobanNetworkConfigForTest(*node);
+    }
     // Now generate Soroban txs
     auto sorobanConfig =
         GeneratedLoadConfig::txLoad(LoadGenMode::SOROBAN, 50,
                                     /* nTxs */ 15, 1, /* offset */ 0);
     sorobanConfig.skipLowFeeTxs = true;
     loadGen.generateLoad(sorobanConfig);
+    auto& loadGenFailed =
+        nodes[0]->getMetrics().NewMeter({"loadgen", "run", "failed"}, "run");
+    auto& secondLoadGenFailed =
+        nodes[1]->getMetrics().NewMeter({"loadgen", "run", "failed"}, "run");
 
     simulation->crankUntil(
         [&]() {
-            return loadGenDone.count() > currLoadGenCount &&
-                   secondLoadGenDone.count() > secondLoadGenCount;
+            return (loadGenDone.count() > currLoadGenCount &&
```

Comment:
Yes, I don't like the timeouts in local tests. They also seem to make the test logs much less readable in CI. So I think that wouldn't hurt to have this condition. FWIW ideally loadgen should probably just have a function like `finished()` that would return true on either success or failure. But that's probably out of scope of this PR.

Reply:
it wouldn't hurt, just adds verbosity (tough on eyes) while yielding the same result as current master. I'd advocate for keeping things tidy (`finished` functions makes sense) as we're stabilizing master, but I suppose this can be cleaned up later if time is tight right now

Reply:
Ok, removed that for the time being.